src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -200,29 +200,53 @@ protected override Conversion GetCollectionExpressionConversion(
                 }
             }
 
+            bool usesKeyValuePairs = CollectionUsesKeyValuePairs(Compilation, collectionTypeKind, elementType, out var keyType, out var valueType);
             var builder = ArrayBuilder<Conversion>.GetInstance(elements.Length);
             foreach (var element in elements)
             {
-                Conversion elementConversion = convertElement(element, elementType, ref useSiteInfo);
-                if (!elementConversion.Exists)
+                switch (element)
                 {
-                    builder.Free();
-                    return Conversion.NoConversion;
+                    case BoundCollectionExpressionSpreadElement spreadElement:
```

Comment:
I expected both the spread case and the expression element case to also be updated with new rules (including for C# 13?) since the element type is a KVP.

Reply:
Implicit item-wise conversions from expression of `KeyValuePair<K1, V1>` type to distinct `KeyValuePair<K2, V2>` type is not included in this PR.